HarpGuru/HarpFace/HarpCell/analysePosition/index.test.ts

```diff
@@ -25,11 +24,11 @@ test('leftmost is true if the provided coord has is 0 in the x axis and false ot
   const yxCoordLeftmostTrue: YXCoord = [0, 0]
   const yxCoordLeftmostFalse: YXCoord = [0, 1]
   const { leftmost: leftmostTrue } = analysePosition({
-    ...harpFaceProps,
+    activeHarpStrata,
```

Comment:
I would like to reserve the `active` prefix for the global state properties so this should be kept in mind with all the upcoming work to improve the various modules using hooks since lots of this functionality may be replaced.

Reply:
The same is true of `activeDisplayMode` and everything else of course.

Reply:
Referenced in open issue: https://github.com/js-jslog/harpnative/issues/45#issuecomment-663620926